### PR TITLE
{chem}[foss/2022b] Dice v20240101

### PR DESCRIPTION
**Pull request by laraPPr**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by laraPPr** [comment]
Test report by @laraPPr
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node4011.donphan.os - Linux RHEL 8.8 (Ootpa), x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 535.129.03, Python 3.11.3
See https://gist.github.com/laraPPr/57700946e4ebb1436b1886a177e201ca for a full test report.

**Comment by laraPPr** [comment]
Test report by @laraPPr
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node4011.donphan.os - Linux RHEL 8.8 (Ootpa), x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 535.129.03, Python 3.11.3
See https://gist.github.com/laraPPr/10d897c8f45ec537f140db06d0c73ac1 for a full test report.

File: easybuild/easyconfigs/d/Dice/Dice-20240101-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+dependencies = [
+    ('Boost.MPI', '1.81.0'),
+    ('HDF5', '1.14.0'),
+]
+
+# Use build environment defined by EB
+prebuildopts = "sed -i 's/^FLAGS_BASE =.*/FLAGS_BASE=$(CXXFLAGS) -g -w -I. $(CPPFLAGS)/' Makefile && "
+buildopts = 'CXX="$MPICXX" USE_INTEL="no" HAS_AVX2="no" '  # avoid changes to -march
+buildopts += 'INCLUDE_MKL="-I${EBROOTOPENBLAS}/include" LIB_MKL="${LIBBLAS}" '  # use OpenBLAS
```
Comment (jfgrimm) [inline easybuild/easyconfigs/d/Dice/Dice-20240101-foss-2022b.eb:38]: we should link `flexiblas` instead of linking `openblas` directly

Comment (boegel) [reply easybuild/easyconfigs/d/Dice/Dice-20240101-foss-2022b.eb:38]:
```suggestion
buildopts += 'INCLUDE_MKL="-I${EBROOTFLEXIBLAS}/include" LIB_MKL="${LIBBLAS}" '  # use FlexiBLAS
```

**Comment by laraPPr** [comment]
@boegelbot please test @ jsc-zen2

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
himem01.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/671475b4da477aa8db3e32064ecd0756 for a full test report.

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @laraPPr!

**Pull request merged**